Sources/MongoSwift/SDAM.swift

```diff
@@ -40,7 +83,11 @@ public struct ServerAddress: Equatable {
 
 extension ServerAddress: CustomStringConvertible {
     public var description: String {
-        "\(self.host):\(self.port)"
+        if self.type == HostType.ip_literal {
```

Comment:
Since we have an enum, it's more common to use a `switch` statement to handle the various cases:
```swift
switch self.type {
case .ipLiteral:
    // do stuff
default:
   // default case
}
```